src/components/inputs/TextInput.tsx

```diff
@@ -7,15 +7,16 @@ import { TextIndication, TextType } from './TextIndication';
 
 interface Props extends TextInputProps {
     style?: ViewStyle;
+    label?: string;
     inputStyles?: TextStyle;
-    label: string;
     value: string;
     text?: string;
     textType: TextType;
     icon?: IconName;
     onChangeText?: (value: string) => void;
     iconSize?: number;
     iconColor?: string;
+    isFocused?: (focused: boolean) => void;
```

Comment:
There is already an `isFocused` property in a `TextInput` to check is the component is focused or no, so it seems weird to override this property in the props for a different purpose, and with a different function signature.

What do you intend to use this property for ?
If you want to have a callback when the focus is acquired or lost, we could use the builtin methods onFocus/onBlur ?

Cf: https://reactnative.dev/docs/textinput#isfocused

Reply:
you are right.
i think for those screens where keyboard is used, i will to use RN TextInput instead of Paper one, because for some reason ref is not working correctly on the Paper Input.

Reply:
Do you have a small reproductible example of what is not working ?

Reply:
if i put ref on TextInput from Paper library, it always returns null for some reason, but when i do the same with the TextInput from RN, it return current ref value

Reply:
You use it to "detect" what TextInput is being focused, is that right ?

Reply:
Yes, i do, but i will change it, because it is not the best way to do it